test/integration/error_handling_test.rb

```diff
@@ -83,31 +85,29 @@ def test_lax_unrecognized_operator
 
   def test_with_line_numbers_adds_numbers_to_parser_errors
     err = assert_raises(SyntaxError) do
-      Liquid::Template.parse(%q(
+      Liquid::Template.parse('
           foobar
 
           {% "cat" | foobar %}
 
           bla
-        ),
-        line_numbers: true
-      )
+        ',
+                             line_numbers: true)
```

Comment:
🤦‍♂ this indentation ...

Reply:
Yeah its a little crazy its lining them up with the original function opening brace.